gnucash/report/reports/standard/category-barchart.scm

```diff
@@ -175,14 +182,21 @@ developing over time"))
       (N_ "Show table")
       "e" (N_ "Display a table of the selected data.")
       #f))
+      
+    (add-option
+     (gnc:make-simple-boolean-option
+      gnc:pagename-display
+      (N_ "Show ratios")
+      "f" (N_ "Display ratios instead of amounts.")
+      #f))
```

Comment:
Instead of using `"f"` and modifying other options's sort strings, it's simpler to use `"e1"` here. Also `Show ratios` is incomplete -- it will no longer draw absolute values in a chart, but will instead report the percentage contribution of each account into the overall asset/liability/income/expense. Is there a better formal name to be used?